components/dash-core-components/tests/integration/dropdown/test_remove_option.py

```diff
@@ -49,7 +52,8 @@ def on_change(val):
     dash_dcc.wait_for_text_to_equal("#value-output", "None")
```

Comment:
@TillerBurr While investigating this issue earlier, I found a quirk of `wait_for_text_to_equal` which gives false positives when used with the text`"None"`.

I recommend to try changing this string to anything besides `"None"` (both here and on line 46) and see whether the test still passes.